lib/index.js

```diff
@@ -103,12 +128,32 @@ export const createStore = (reducer, initialState, enhancer) => {
  * @returns {Function} - A function that expects two arguments: an Object of key/value pairs where the values are
  *                       Backbone entities, and an optional store enhancer. Returns a store.
  */
-export const bbCreateStore = (storeCreator = createStore) => (backboneEntities, enhancer = noop) => {
-    const combinedBBEntities = bbCombineEntities(backboneEntities);
+export const bbCreateStore = (storeCreator = createStore) => (bbEntityOrReducer, enhancer = noop) => {
+    let bbReturningReducer;
+    let combinedBBEntities;
+    let passedBBEntity = typeof bbEntityOrReducer !== 'function';
```

Comment:
I'm open to be convinced on the API usage, but the goal was to create a way
that you can update an existing model/collection without actually modifying
it.  It's the middle-layer thing that responds to the redux actions without
having to handle all of them internally to the bb entity.

Do you not feel that would be something someone would like to do?  I would
agree, however, that it is not the primary/basic usage.

On Fri, Aug 19, 2016 at 5:22 PM, Vitor Balocco <notifications@github.com>
wrote:

> In lib/index.js
> <https://github.com/adamterlson/backbone-redux-store/pull/6#discussion_r75559864>
> :
>
> > @@ -103,12 +128,32 @@ export const createStore = (reducer, initialState, enhancer) => {
> >   * @returns {Function} - A function that expects two arguments: an Object of key/value pairs where the values are
> >   *                       Backbone entities, and an optional store enhancer. Returns a store.
> >   */
> > -export const bbCreateStore = (storeCreator = createStore) => (backboneEntities, enhancer = noop) => {
> > -    const combinedBBEntities = bbCombineEntities(backboneEntities);
> > +export const bbCreateStore = (storeCreator = createStore) => (bbEntityOrReducer, enhancer = noop) => {
> > +    let bbReturningReducer;
> > +    let combinedBBEntities;
> > +    let passedBBEntity = typeof bbEntityOrReducer !== 'function';
>
> This can be const since you never reassign it
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/adamterlson/backbone-redux-store/pull/6/files/6405c1fd44311b72ff41daabea06bba4de6664a2#r75559864>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACYfcvSaWt4YxZm4ThKyQbb_D5N03v3fks5qhiyKgaJpZM4Jo4YV>
> .
>